dockers/docker-platform-monitor/docker-pmon.supervisord.conf.j2

```diff
@@ -41,6 +41,17 @@ dependent_startup=true
 dependent_startup_wait_for=rsyslogd:running
 {% endif %}
 
+[program:chassisshow]
```

Comment:
I am thinking this is not a daemon, just a one-shot task, so maybe "docker_init.sh" is a better place to start it.

Reply:
The reason this is not in docker_init.sh is that it is not a configuration / environment related task, it is a process that makes platform API calls and then makes STATE_DB calls. 

There seems to be a clear break with the dockers of the init script handling environment / config gen / etc. and supervisord handling the execution of any programs making database calls (doing the "actual work").

I also think having this in supervisord makes this more maintainable because it contains all of the programs a developer would need to examine to see what is calling the platform API and impacting the database as part of the platform monitor. 

I believeI have configured the supervisord settings correctly but if I have not I am glad to make any changes that make it more correct for a one-shot task.